packages/mobile/src/localCurrency/SelectLocalCurrency.tsx

```diff
@@ -11,51 +15,50 @@ import { navigateBack } from 'src/navigator/NavigationService'
 
 const DEFAULT_CURRENCY_CODE = LocalCurrencyCode.USD
 
-const keyExtractor = (item: LocalCurrencyCode) => item
-
 function SelectLocalCurrency() {
   // tslint:disable-next-line: react-hooks-nesting
   const selectedCurrencyCode = useLocalCurrencyCode() || DEFAULT_CURRENCY_CODE
   const dispatch = useDispatch()
+  const { t } = useTranslation(Namespaces.accountScreen10)
 
-  const onSelect = useCallback(
-    (code) => {
-      dispatch(selectPreferredCurrency(code))
-
-      // Wait for next frame before navigating back
-      // so the user can see the changed selection briefly
-      requestAnimationFrame(() => {
-        navigateBack()
-      })
-    },
-    [dispatch]
-  )
-
-  const renderItem = useCallback(
-    ({ item: code }) => (
-      <SelectionOption
-        text={code}
-        onSelect={onSelect}
-        isSelected={code === selectedCurrencyCode}
-        data={code}
-      />
-    ),
-    [selectedCurrencyCode]
-  )
+  function onSelect(code: string) {
+    dispatch(selectPreferredCurrency(code as LocalCurrencyCode))
 
+    // Wait for next frame before navigating back
+    // so the user can see the changed selection briefly
+    requestAnimationFrame(() => {
+      navigateBack()
+    })
+  }
   return (
-    <FlatList
-      style={styles.container}
-      data={LOCAL_CURRENCY_CODES}
-      extraData={selectedCurrencyCode}
-      renderItem={renderItem}
-      keyExtractor={keyExtractor}
-    />
+    <ScrollView style={styles.container}>
+      <SafeAreaView>
```

Comment:
Don't we usually put SAV outside the scrollview?

Reply:
Actually, @jeanregisser  pointed me out against it - otherwise we see padding at the bottom, when scrolling, which is not native experience.